src/types.cr

```diff
@@ -32,6 +32,10 @@ struct Int
       io << ""
     end
 
+    def inspect
```

Comment:
Isn't there a generic way of inspecting this Null objects ( something like a macro in top level )